package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
Good work, can you add `--coverage` flag to view the coverages across all files,  and also an extra script for continuous using the `--watchAll` flag `eg "test:watch" : "jest --watchAll --coverage"` this would be useful when running test and make use of environments when testing that would differentiate between testing and development environment.

Reply:
@davidokonji, the option to view coverage has been set already in the jest config file. The script for test:watch would be included accordingly.

Reply:
fixed @davidokonji